Directory.Build.props

```diff
@@ -21,7 +20,8 @@
     <EnableSourceLink Condition=" '$(OS)' != 'Windows_NT' AND '$(MSBuildRuntimeType)' != 'Core' ">false</EnableSourceLink>
 
     <!-- suppress false positive warning FS2003 about invalid version of AssemblyInformationalVersionAttribute -->
-    <NoWarn>FS2003</NoWarn>
+    <!-- Supress NU5105 triggerd by MinVer height e.g.: pr43-rc1.2: The package version '<X>' uses SemVer 2.0.0 or components of SemVer 1.0.0 that are not supported on legacy clients. Change the package version to a SemVer 1.0.0 string. If the version contains a release label it must start with a letter. This message can be ignored if the package is not intended for older clients. -->
+    <NoWarn>FS2003;NU5105</NoWarn>
```

Comment:
It's better to do `<NoWarn>$(NoWarn);FS2003;NU5105</NoWarn>` otherwise you risk overwriting values which have been set elsewhere.